api/src/main/java/org/openmrs/api/cache/OpenmrsCacheManagerFactoryBean.java

```diff
@@ -18,6 +18,10 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * This class creates cache configurations from apiCacheConfig.properties files in the classpath. If the configuration
+ * already exist it won't be overridden
```

Comment:
Please paste an example of apiCacheConfig.properties content here as well and say it is for use in modules.